.travis.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - stage: tag
       language: generic
       # no install necessary for generic language
-      install: false
+      install: true
```

Comment:
is this necessary?

Reply:
Yea, build was failing. I think we can just remove it, but for now i left it as true like it was in another section